lib/chatbot-api/chatbot-s3-buckets/index.ts

```diff
@@ -39,7 +40,31 @@ export class ChatBotS3Buckets extends Construct {
       ],
     });
 
+    const userFeedbackBucket = new s3.Bucket(this, "UserFeedbackBucket", {
+      blockPublicAccess: s3.BlockPublicAccess.BLOCK_ALL,
+      removalPolicy: cdk.RemovalPolicy.DESTROY,
+      autoDeleteObjects: true,
+      transferAcceleration: true,
+      enforceSSL: true,
+      serverAccessLogsBucket: logsBucket,
+      cors: [
```

Comment:
Why CORS are enabled here? We are not dealing with objects directly from web clients